numba/cuda/tests/cudapy/test_constmem.py

```diff
@@ -168,24 +167,10 @@ def test_const_record_optimization(self):
         B = np.zeros(2, dtype=int)
         jcuconst = cuda.jit(cuconstRec).specialize(A, B)
 
-        rtver = cuda.runtime.get_version()
-        old_runtime = rtver in ((9, 0), (9, 1))
-        nvvm70_runtime = rtver >= (11, 2)
+        nvvm70_runtime = cuda.runtime.get_version() >= (11, 2)
         windows = sys.platform.startswith('win')
 
-        if old_runtime:
-            if windows:
-                # For some reason Win64 / CUDA 9.1 and 9.2 decide to do two u32
```

Comment:
9.2 is mentioned in the comments here, should this branch be removed?

Reply:
The comment was wrong and should have read "9.0 and 9.1" - the version check above is `rtver in ((9, 0), (9, 1))`.

Reply:
ah yes, thanks, removal seems doubly valid!